cfme/tests/pod/test_appliance_crud.py

```diff
@@ -249,8 +325,7 @@ def test_crud_pod_appliance_ansible_deployment(temp_pod_ansible_appliance, provi
     assert navigate_to(proj, 'Dashboard')
 
 
-@pytest.mark.manual
-def test_crud_pod_appliance_ext_db():
+def test_crud_pod_appliance_ext_db(temp_extdb_pod_appliance, provider, setup_provider):
```

Comment:
Optional:
It look like all test required `setup_provider` in this file... not related to this PR but if you like you can move to global mark

Reply:
I'm gonna add more tests. So, I can't predict if new tests will need setup_provider. So, I'd leave it as is.